### ISSUE TITLE
Option to postpone draft backup?

### ISSUE THREAD

**Issue by wmjordan** [jgraph/drawio-desktop#849]
* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
I monitored the file system with Process Monitor how draw-io desktop worked.
I found that after each operation on the editing document, draw-io appended data to a log file within _AppData\Roaming\draw.io\IndexedDB\file__0.indexeddb.leveldb_.
I opened that log file and found that the appended data appeared to be the whole file of the editing document and that file incrementally got larger and larger after edits. The larger the editing document, more data got appended to the log file. Undo operation also made that file get larger.

**Describe the solution you'd like**
Could you provide an option to limit the auto-backup feature to execute only after a specific minutes?
For instance, if the value is 5, the auto-backup only executes 5 minutes after the last backup and new modifications have occurred within the last 5 minutes.

**Additional context**
Version: 16.0.2.
I've disabled auto-save feature.

**Comment by davidjgraph** [comment]
The issue here is more this doesn't scale, regardless of delay.

**Comment by m-mohamedin** [comment]
@wmjordan The draft is deleted when the file is saved or program is closed normally. Does the size of the Indexed DB folder keeps growing between sessions?
On Mac, the Indexed DB folder is not growing between sessions

**Comment by wmjordan** [comment]
Nope. The aforementioned file in _file__0.indexeddb.leveldb_ folder was not removed.
I opened a file, made some modifications, then closed draw-io with "discard changes".
The file was still there after that and changes were appended to the file continually.

And I don't need draw-io to backup the document after each modification or undo operation. The current backup is too frequent for me.
